### PR TITLE
Clone s24

### PR DESCRIPTION
**Pull request by Riaboshtan**

### CODE REVIEW

File: apps/user_profile/forms.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+from django import forms
+from django.views.generic import FormView
+
+from apps.user_profile.models import Child
+
+
+class RegisterChildForm(forms.ModelForm):
+        class Meta:
+            model = Child
+            fields = ('first_name', 'last_name', 'surname', 'dob', 'sex', 'city', 'street', 'house', 'flat',
```
Comment (aodarc) [inline apps/user_profile/forms.py]: so many fields in list)))
refactor that like this https://docs.djangoproject.com/en/2.0/topics/forms/modelforms/#selecting-the-fields-to-use
You should to add 
`fields = '__all__'`
`exclude = ['some_fields']`

File: apps/user_profile/forms.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                      'birth_sertificate_series', 'birth_sertificate_number', 'birth_sertificate',
+                      'privilege', 'privilege_series', 'privilege_number')
+
+
+    # last_name = forms.CharField( max_length=100)
```
Comment (aodarc) [inline apps/user_profile/forms.py]: also remove all comments except docstrings

File: apps/user_profile/forms.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+)
+
+
+class RegisterChildView(Form):
+
+    lastname = CharField(
+        label='Прізвище дитини',
+        max_length=100,)
+    name = CharField(
+        label='Ім\'я дитини' ,
```
Comment (aodarc) [inline apps/user_profile/forms.py]: What did I see? `\`
if you use " " (double quotes) then you do not need shield the apostrophe

File: apps/user_profile/views.py [hunk 4]
```diff
@@ -42,3 +43,16 @@ class RegisterFormView(FormView):
...
         form.save()
         return super(RegisterFormView, self).form_valid(form)
+
+
+def child_form(request):
+    if request.method == 'POST':
+        form = RegisterChildForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect('/success/')
```
Comment (aodarc) [inline apps/user_profile/views.py:53]: do you have `success` url?

File: apps/user_profile/models.py [hunk 5]
```diff
@@ -91,6 +92,59 @@ class Child(models.Model):
         blank=False,
         verbose_name='Стать'
     )
+
+    city = models.CharField(
```
Comment (aodarc) [inline apps/user_profile/models.py]: this is should be FK to https://github.com/aodarc/final_project/blob/0481c12b365f32a14124b9d4ff270ca83cdbc70c/apps/location/models.py#L15

**Review by aodarc — CHANGES_REQUESTED** [review]

File: apps/user_profile/views.py (hunk 4 above)
Comment (Riaboshtan) [reply apps/user_profile/views.py:53]: it's not like url, after successful registration it shows pop-up that redirect to user page

**Pull request closed without merging**